### ISSUE TITLE
remove_chef_system_gem recipe removed in 3.0, but still being called from default

### ISSUE THREAD

**Issue by signe** [chef-cookbooks/omnibus_updater#140]
```
Chef::Exceptions::RecipeNotFound
--------------------------------
could not find recipe remove_chef_system_gem for cookbook omnibus_updater

Cookbook Trace:
---------------
  /var/chef/cache/cookbooks/omnibus_updater/recipes/default.rb:34:in `from_file'
  ...
```

**Comment by signe** [comment]
Broken by #134 unnecessarily.  If users need to remove the system gem why would you pull this functionality?

**Comment by tas50** [comment]
I've removed the block from default.rb and released 3.0.1. 

I removed this functionality as for almost every user their installation is an omnibus install, which is the only installation method supported by Chef. Chef installed in the system rubycan either remain or be removed with a simple gem resource anywhere in the run list.
